sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -641,11 +660,12 @@ export class SecretClient {
 
   private async *listPropertiesOfSecretVersionsAll(
     secretName: string,
-    options?: ListOperationOptions
+    options: ListOperationOptions = {}
   ): AsyncIterableIterator<SecretProperties> {
+    const requestOptions = operationOptionsToRequestOptionsBase(options);
```

Comment:
Probably these private methods should be updated to take a ROB? Else, the options should be passed as-is from the public methods? Right now the options are getting double-converted I believe. This will have no effect now, but could be problematic in the future as this interface grows.

Reply:
I'll fix it

Reply:
It should be fixed now :)